components/br-stream/src/observer.rs

```diff
@@ -229,6 +233,16 @@ mod tests {
 
     use super::BackupStreamObserver;
 
+    macro_rules! assert_let {
```

Comment:
wait i've seen this in another PR 🤔

Reply:
Yes, that PR is actually extract from this PR.